### ISSUE TITLE
Cairo headers in OS X

### ISSUE THREAD

**Issue by gaperez64** [xournalpp/xournalpp#2918]
**Is your feature request related to a problem? Please describe.**
The Homebrew build steps currently **almost** work for OS X on M1 chips. The only issue is that cairo headers imported in the files listed below refer to `cairo/*.h` instead of the header directly. (That is, they name the header in a directory cairo.)
- src/control/PdfCache.h
- src/pdf/base/XojCairoPdfExport.cpp
- src/pdf/base/XojPdfPage.h
- src/view/PdfView.h

To get the code to compile, I have to remove the directory part of the imported headers.

**Describe the solution you'd like**
I think this is just a problem with the pkg-config files for cairo in Linux compared to OS X. When calling `pkg-config cairo --cflags` I get the following include library added `-I/opt/homebrew/Cellar/cairo/1.16.0_5/include/cairo`. I expect that on linux the last directory is not there? This should be fixable using CMake (and removing the directory in the files), maybe just add more include directories by appendix `/cairo` to the ones already added? This would make it work for both Linux and OS X.

**Comment by rolandlo** [comment]
On Ubuntu 20.10 the output of `pkg-config cairo --cflags` is 
```
-I/usr/include/cairo -I/usr/include/glib-2.0 -I/usr/lib/x86_64-linux-gnu/glib-2.0/include -I/usr/include/pixman-1 -I/usr/include/uuid -I/usr/include/freetype2 -I/usr/include/libpng16
```

**Comment by gaperez64** [comment]
And if you check the contents of `/usr/include/cairo` do you have a directory named `cairo` or the headers directly there?

**Comment by rolandlo** [comment]
They are directly in that directory:

```
roland@roland-ThinkPad-X1-Yoga-3rd:~$ ls /usr/include/cairo
cairo-deprecated.h  cairo-ft.h       cairo.h      cairo-ps.h      cairo-script-interpreter.h  cairo-tee.h      cairo-xcb.h   cairo-xlib-xrender.h
cairo-features.h    cairo-gobject.h  cairo-pdf.h  cairo-script.h  cairo-svg.h                 cairo-version.h  cairo-xlib.h
```

**Comment by rolandlo** [comment]
Actually if I replace `cairo/*.h` by `*.h` in these 4 files, Xournal++ still builds fine (on Ubuntu 20.10).

**Comment by rolandlo** [comment]
Maybe the fix is simply #2919? It builds on Linux, Mac and Windows.

**Comment by gaperez64** [comment]
Yup, based on your answers, I was going to propose the same.
